admin/DumpJSON

```diff
@@ -28,7 +28,7 @@ Interally, when making the requests to fetch the JSON, all inc parameters are
 included except:
 
  * Those that perform subqueries and return 25 linked entities:
-   https://wiki.musicbrainz.org/Development/XML_Web_Service/Version_2#Subqueries
+   https://wiki.musicbrainz.org/MusicBrainz_API#Subqueries
```

Comment:
Why? I mean, the page just moved so we move that, there's nothing else to say in the commit message IMO since that's not connected to the actual code at all :) I did mention the moving (and why) on the ticket now though!

Reply:
Ticket is fine too, thanks.